Mathlib/Analysis/Convex/StrictConvexBetween.lean

```diff
@@ -16,10 +16,25 @@ space.
 
 -/
 
+open Metric
+open scoped Convex
 
+variable {V P : Type*} [NormedAddCommGroup V] [NormedSpace ℝ V]
+
+section to_move
```

Comment:
I don't have a suggestion for where to move this to; does `#find_home` help?

Either way, I think we should drop `section to_move` before merging this.